Rasa_Bot/endpoints.yml

```diff
@@ -11,7 +11,7 @@
 # https://rasa.com/docs/rasa/custom-actions
 
 action_endpoint:
-  url: "http://localhost:5055/webhook"
+  url: "http://rasa_actions:5055/webhook"
```

Comment:
The root docker compose no longer works for me with this change. The custom actions cannot be executed.

Reply:
We resolved this together, it does actually work!